documentation/docs/11-ssr-and-javascript.md

```diff
@@ -22,6 +22,8 @@ You can disable SSR app-wide with the [`ssr` config option](#configuration-ssr),
 </script>
 ```
 
+Note: `"never"` is not supported on page level `ssr` exports.
```

Comment:
I suggest adding a sentence to explain that if the `ssr` config option is set to `"never"` then page-level `ssr` exports are ignored.